src/modules/flow/iio/iio.json

```diff
@@ -822,6 +822,12 @@
              "name": "sampling_frequency"
          },
          {
+             "data_type": "string",
+             "default": null,
+             "description": "Channel name. If none, try to auto detect channel",
+             "name": "channel_name"
```

Comment:
Thanks @edersondisouza. I got your point. Just change the reader_cb_data to a list, and store all these config->data. Please check the new PR. YES, one single tick will output 4 RAW data:
adc1:698.242188 (float range)
adc2:3696.289062 (float range)
adc1:805.664062 (float range)
adc2:3608.398438 (float range)